src/NHSD.BuyingCatalogue.Ordering.Domain/CataloguePriceType.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NHSD.BuyingCatalogue.Ordering.Domain
+{
+    public sealed class CataloguePriceType
+    {
+        public static readonly CataloguePriceType Flat = new CataloguePriceType(1, nameof(Flat));
+        public static readonly CataloguePriceType Tiered = new CataloguePriceType(2, nameof(Tiered));
+
+        private CataloguePriceType(
+            int id, 
```

Comment:
Why are the parameters on seperate lines? Wouldnt it make sense for them to all be on one, as I doubt that will break the line limit

Reply:
Just found it easier to read. I will change them.